src/component/BooksRender.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RemoveBooks from './Removebook';
+
+const Books = ({ id, title, author }) => (
+  <div className="bookSite">
+    <div className="info">
+      <div className="left-section">
+        <ul>
+          <li className="genre books">Action</li>
+          <div>
+            <li className="booktitle books">{title}</li>
+          </div>
+          <li className="bookauthor books">{author}</li>
+        </ul>
+
+        <div className="book-actions">
+          <button className="book-btn" type="button">
+            Comments
+          </button>
+          <span>|</span>
+          <RemoveBooks id={id} />
```

Comment:
- [ ] I think here you can fix the alignment here since I can see a big gap between the text and buttons (edit, remove, update) so please make sure to fix the spaces as much as possible ✨🚀🔥💯


- Your design
![image](https://user-images.githubusercontent.com/58553711/204062810-6e69e68b-46fe-4187-91e1-3b4d1f8fd437.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/204062824-84a30932-6b1e-4ffd-bd8f-1e3412426784.png)

Reply:
fixed